repl.html

```diff
@@ -120,38 +120,45 @@
     input.addEventListener("keyup", e => inputHandler(e));
 }
 
+async function submitInput() {
+    const response = await sendRequest(document.env, input.value);
+    if (response.ok) {
+        const {env, result} = await response.json();
+        updateHistory(input.value, result);
+        history.push([input.value, result]);
+        input.value = "";
+        document.env = env;
+        window.localStorage.setItem('env', env)
+        window.localStorage.setItem('history', JSON.stringify(history));
+    } else {
+        const msg = await response.text();
+        updateHistory(input.value, msg);
+        input.value = "";
+    }
+}
+
 loadFromLocalStorage();
 input.focus();
 async function inputHandler(e) {
     // TODO(max): Make up/down arrow keys navigate history
     if (e.key === "Enter") {
-        if (e.shiftKey && input.tagName === "INPUT") {
+        if (input.tagName === "INPUT" && e.shiftKey) {
```

Comment:
Can you put this back how it was

Reply:
I did this to match the structure of the textarea conditional below. Can put it back if you feel strongly.

Reply:
Eh, not necessary for the diff. I tend to leave as-is as much as possible (leaving stylistic changes to their own commit(s))

Reply:
Makes reverting easy

Reply:
I don't care a lot for this in particular either way

Reply:
Alright, as you wish